### ISSUE TITLE
weird style choices

### ISSUE THREAD

**Issue by mcandre** [jessfraz/dockfmt#21]
Why does dockfmt add pointless tabs after Docker keywords?

Why does dockfmt insert a blank line at the start of the Dockerfile?

**Comment by blez** [comment]
Agree. The current style is unusable. It would be nice to have a flags that allow to have my own style of formatting, like in yamlfmt or shfmt

**Comment by vladdoster** [comment]
@mcandre @blez,

As the README.md states, it is modeled after `gofmt` which also uses tabs so I assume that is why tabs are used.

You can fork the project and change/add things that you want. Here is [the line that adds a tab between the directive and text](https://github.com/jessfraz/dockfmt/blob/1455059b8bb53ab4723ef41946c43160583a8333/format.go#L165).

---

Tangential, but I've been searching something better than [`checkmake`](https://github.com/mrtazz/checkmake) and  [`unmake`](https://github.com/mcandre/unmake) scratches that itch. Thanks, @mcandre.
